src/Illuminate/Console/Application.php

```diff
@@ -55,11 +55,13 @@ public function __construct(Container $laravel, Dispatcher $events, $version)
      */
     public function call($command, array $parameters = [])
     {
-        $parameters['command'] = $command;
+        $parameters = collect($parameters)->prepend($command);
 
         $this->lastOutput = new BufferedOutput;
 
-        return $this->find($command)->run(new ArrayInput($parameters), $this->lastOutput);
+        $this->setCatchExceptions(false);
```

Comment:
no, the Kernel::handle() calls getArtisan()->run() which deffers to the symfony application method which calls doRun() and so on (allowing for adding of a console dispatcher, finding the command and running it).

calling Artisan::call() simply fetches the command from the application and runs it (without going through the symfony console run() method), hence no console dispatcher can be added.

by adding a dispatcher you can listen to events when a command is run, disable the command and perform tasks before and after its run. This isnt possible with Artisan::call.